Dockerfile

```diff
@@ -50,16 +50,17 @@ RUN npm run database:generate-typings
 
 # Image to run, copy everything needed from builder
 FROM node:20-slim
-
 LABEL org.opencontainers.image.source="https://github.com/ghostfolio/ghostfolio"
+ENV NODE_ENV=production
 
-RUN apt update && apt install -y \
-  curl \
-  openssl \
-  && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y --no-install-suggests \
+    curl \
```

Comment:
Can you keep the indentation?

Reply:
fixed in https://github.com/ghostfolio/ghostfolio/pull/3614/commits/fd3823a70dcdb1c32e43478b955bbb2fc66ef6b3